pkg/project/bind.go

```diff
@@ -152,15 +155,18 @@ func Bind(projectPath string, name string, language string, projectType string,
 	return &response, nil
 }
 
-func completeBind(projectID string, conURL string) (string, int) {
-	uploadEndURL := conURL + "projects/" + projectID + "/bind/end"
+func completeBind(projectID string, conURL string, username string, connection *connections.Connection) (string, int) {
+	uploadEndURL := conURL + "/api/v1/projects/" + projectID + "/bind/end"
 
 	payload := &BindEndRequest{ProjectID: projectID}
 	jsonPayload, _ := json.Marshal(payload)
 
 	// Make the request to end the sync process.
-	resp, err := http.Post(uploadEndURL, "application/json", bytes.NewBuffer(jsonPayload))
-	if err != nil {
+	request, err := http.NewRequest("POST", uploadEndURL, bytes.NewBuffer(jsonPayload))
+	request.Header.Set("Content-Type", "application/json")
+	resp, httpSecError := sechttp.DispatchHTTPRequest(http.DefaultClient, request, username, connection.ID)
+
+	if httpSecError != nil {
 		panic(err)
```

Comment:
Shouldn't being using a panic to catch the error as it throws the whole stack. Not part of this PR and out of scope for a change but needs to be addressed.

Reply:
Agreed - should be addressed in a new PR